pkg/specprovider/specprovider.go

```diff
@@ -46,6 +48,17 @@ func (ppb *specProviderBuilder) New(client client.Client, builder configmanager.
 			return nil, err
 		}
 		return mgr, nil
+	case "LOCAL":
```

Comment:
For this and the other various similar comparisons, I'm starting to think we should maybe change the value evaluated by the `switch` to a `strings.ToUpper()`-ized equivalent first, so that there are no case sensitivity issues if `"local"` is used.

Probably should do it for the upgradeconfig upgrade type too, whilst I'm thinking of it.